CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.14.4
+
+## Release v1.14.4 - 2022/01/06
+
+### Enhancement
+
+* in_tail: add option to skip long lines (max_line_size)
+  https://github.com/fluent/fluentd/pull/3565
+
+### Bug fix
```

Comment:
Please add #3560 too

Reply:
It is useful if we could explain a bit more. (when it should be used)

Reply:
I added the item to the release note in d1f33da.